style.css

```diff
@@ -65,12 +74,236 @@ header a {
   column-gap: 20px;
 }
 
+button:hover {
+  cursor: pointer;
+}
+
+.Works_Section {
+  display: grid;
+  grid-template-columns: 80%;
+  grid-template-rows: 24% 24% 24% 24%;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 88px 24px 88px 24px;
+}
+
+.card_1 {
+  height: 552px;
+}
+
+.card_2 {
+  height: 596px;
+}
+
+.card_3 {
+  height: 552px;
+}
+
+.card_4 {
+  height: 596px;
+}
+
+.card img {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  margin: 16px 16px 0 16px;
+}
+
+.card h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 12px 16px 0 16px;
+}
+
+.frame ul {
+  display: flex;
+  margin: 12px 16px 12px 16px;
+  column-gap: 12px;
+  align-items: center;
+
+  /* background-color: red; */
+}
+
+.frame li {
+  list-style: none;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.separator {
+  width: 6px;
+  height: 6px;
+  border-radius: 16px;
+  background-color: #c1c7d0;
+}
+
+.frame .canopy {
+  list-style: none;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.card p {
+  margin: 0 16px 0 16px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tags ul {
+  display: flex;
+  margin: 12px 16px 12px 16px;
+  column-gap: 12px;
+
+  /* align-items: center; */
+}
+
 .social-media-icons li {
+  display: flex;
+  padding-left: 12px;
+  margin-bottom: 12px;
+  column-gap: 12px;
+
+  /* align-items: center; */
+}
+
+.categories li {
   list-style: none;
 }
+
+.tags ul li {
+  list-style: none;
+  margin-top: 0;
+  padding-top: 0;
+}
+
+.tags button {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border-radius: 8px;
+  background: #ebebff;
+  border: none;
+  padding: 4px 12px 4px 12px;
+}
+
+.see_project_button button {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  margin-left: 16px;
+}
+
+.content_2 {
+  width: 375px;
+  height: 5000px;
+  height: 674px;
+  background: #fff;
+  border-top-right-radius: 60px;
+}
+
+.content_2 h2 {
+  padding: 114px 24px 0;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.content_2 p {
+  padding: 12px 24px 12px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.content_2 h6 a {
+  text-decoration: none;
+  padding: 0 24px 12px;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.resume_button button {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  margin-left: 24px;
+}
```

Comment:
[OPTIONAL]

- [ ] The same remark for this button as well please add styling to the states of the button. 😁